### ISSUE TITLE
Odd placement/sizing of container details pane

### ISSUE THREAD

**Issue by deboer-tim** [podman-desktop/podman-desktop#82]
### Bug description

Short summary: I'm picky and the current pane showing logs and terminal isn't working for me. :)

- I didn't know this pane existed until I clicked on a container. Clicking made the left tabs (Containers, Images) shrink, and moved the Create container button half way off screen with no way to scroll it back (see image below). These things 'outside' of the container section shouldn't move.
- The details pane itself doesn't fit in the default window size and no way to scroll right. Yes, I can make the window bigger, but maybe I don't want to or don't realize there are buttons hiding off to the right.
- IMHO we don't need a terminal button since I can click on the tab at the bottom.
- The container name and image are duplicated from what was in the tab already (see comment below).
- The Command column is too narrow and wrapped (and Port in another case), should use more of the pane width for the table.
- "Logs of wizardly_roentgen will appear there if any..." is a bit wordy, maybe just "Log output will appear here..."
- It wasn't clear to me how to close the pane, clicking on white-space didn't work (i.e. clicking on the container on the left again wasn't obvious enough).
- If containers have a detail pane I expect images would have it too. This may be in plan, just saying it should be consistent.

I'd prefer it if the details pane took up the entire size of the container tab (ie replace the containers list), with a back button to the left of the container image (or close button at top right). This would make better use of space and avoid many of the issues above.

<img width="1154" alt="Screen Shot 2022-04-05 at 11 10 40 AM" src="https://user-images.githubusercontent.com/19958075/161786279-13139672-951c-405c-9f5f-09897d1de7c9.png">

### Operating system

masOC 12.3.1 M1

### Version

next (development version)

**Comment by benoitf** [comment]
cc @slemeur 

about the biggest complaint about the 'layout' I think the reason of not wiping out the list to display details is to be able to toggle the container with one click

Other details could be fixed but it might depend on how to address the major complaint

**Comment by slemeur** [comment]
@deboer-tim: 
The idea with the layout and keeping the list of containers on the left was that it allows to easily switch from one container to another one without leaving the context. 

With a bit of UI work that screen would look like this:
<img width="966" alt="Capture d’écran 2022-04-14 à 11 47 10" src="https://user-images.githubusercontent.com/1636769/163360094-f7d7fab2-8799-4789-b873-2350b23e6764.png">

**Comment by deboer-tim** [comment]
Yeah, I understand, and the ability to switch is nice. The problem is that I keep the window at default size (about 1/2 width and 1/2 height of my screen, which I like) so the initial experience felt more like this:

<img width="720" alt="smaller" src="https://user-images.githubusercontent.com/19958075/163388180-0a81a245-9bd8-48dd-8e57-da69d72def18.png">

If I care about the logs or terminal window, they're only a tiny portion of my screen. There's a lot of empty space in the first two columns, and (in the current implementation) most of the information in the second column gets cutoff with '...' to make room, and is duplicated in the right side.

Personally, I would give up the ability to quickly switch for a usable terminal window. If you disagree, then I think we need to look at a few other steps to save space, e.g. reducing the first column to icons, making the second column narrower when viewing the terminal, or an action to temporarily hide the second column or make the terminal window full screen.

**Comment by slemeur** [comment]
This will be addressed as part of the effort on #143
